spec/models/hydrus/generic_object_spec.rb

```diff
@@ -399,6 +399,7 @@ def genre
 
     context 'when configured to start common assembly' do
       it 'invokes the client to publish' do
+        allow(@go).to receive(:id).and_return('druid:oo000oo0003')
```

Comment:
Should this be `:pid`?

Reply:
That's one of the changes I just pushed.